security/permissions/src/main/java/io/quarkus/ts/openshift/security/basic/permissions/resources/ClassAnnotationPermissionAllowedInclusiveResource.java

```diff
@@ -0,0 +1,27 @@
+package io.quarkus.ts.openshift.security.basic.permissions.resources;
+
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+
+import io.quarkus.security.PermissionsAllowed;
+
+@Path("/class-permission-allowed")
+@PermissionsAllowed(value = { "read:all", "create" }, inclusive = true)
+public class ClassAnnotationPermissionAllowedInclusiveResource {
+
+    public static final String RESPONSE_STRING = "Permitted";
```

Comment:
Maybe it can be changed to a different value, such as _"PermittedAllowedInclusive"_? To differentiate from other resource classes?

Reply:
I changed it for endpoint which are inherting the class permissions. In general I just check if the endpoint show some text and not just `200 OK` response